app/resources/create.html.template

```diff
@@ -94,6 +115,14 @@
               </div>
             {% endif %}
 
+            <a href="#" class="identify-button" style="text-decoration:none">
+              {% trans "+ Other personal identify detail" %} <br>
```

Comment:
already edited to 
+ More information to identify the person
+ More information about source of record
+ More information about status of the person

is it better?